Neos.ContentRepository.BehavioralTests/Tests/Behavior/Features/NodeCopying/CopyNode_NoDimensions.feature

```diff
@@ -65,3 +67,24 @@ Feature: Copy nodes (without dimensions)
       | nodeAggregateIdMapping                 | {"sir-nodeward-nodington-iii": "sir-nodeward-nodington-iii-copy"} |
 
     Then I expect node aggregate identifier "sir-nodeward-nodington-iii-copy" to lead to node cs-identifier;sir-nodeward-nodington-iii-copy;{}
+
+  Scenario: Copy References
```

Comment:
nice :D and does it also work across dimensions? see https://github.com/neos/neos-development-collection/issues/4614 ...it should work right?